action.yml

```diff
@@ -45,13 +68,22 @@ runs:
     - name: Run OtterAI Code Review
       shell: bash
       env:
+        INPUT_PROVIDER: ${{ inputs.provider }}
```

Comment:
🔧 Ensure that all environment variables are correctly set and used in the script to avoid runtime errors.